app/Http/Resources/V1/ActivityResource.php

```diff
@@ -14,7 +15,22 @@ class ActivityResource extends JsonResource
      * @return array
      */
     public function toArray($request)
-    {
+    {        
+        $h5p_parameters = $user_name = $user_id = null;
+        
+        if($this->playlist->project->user){
+            $user_name = $this->playlist->project->user;
+            $user_id = $this->playlist->project->id;
+        }
+        if($this->type === 'h5p'){
```

Comment:
Please add spaces before and after if condition.

Reply:
updated